R/zchunk_LA143.HDDCDD.R

```diff
@@ -118,10 +118,18 @@ module_energy_LA143.HDDCDD <- function(command, ...) {
         mutate(iso = if_else(iso == "scg", "mne", iso))
     }
 
+    # Extend population data to all years
+    iso_list <- tibble(iso = L101.Pop_thous_GCAM3_ctry_Y$iso %>% unique())
+    all_years <- tibble(year = seq(1971, 2100))
```

Comment:
These numbers should probably come from `min(HISTORICAL_YEARS)` and `max(FUTURE_YEARS)`, rather than be hardcoded.